requirements/edx/base.txt

```diff
@@ -120,6 +120,7 @@ django_debug_toolbar==1.3.2
 
 # Used for testing
 astroid==1.3.8
+before_after==0.1.3
```

Comment:
See https://github.com/edx/edx-platform/pull/10340 for details of this library being added.